app/client/src/features/events/EventSidebar/EventSidebar.tsx

```diff
@@ -154,17 +155,25 @@ export const EventSidebar = ({ fragmentRef }: EventSidebarProps) => {
             {!data.isViewerModerator && <QuestionCarousel fragmentRef={data} />}
 
             {data.isViewerModerator && (
-                <Tabs variant='scrollable' scrollButtons='auto' value={tabIndex} onChange={handleTabChange}>
-                    <Tab label='Question Queue' />
-                    <Tab label='Question List' />
-                    <Tab label='Live Feedback' />
-                </Tabs>
+                <Tabs 
+                    value={tabIndex}
+                    onChange={handleTabChange}
+                    tabs={[
+                        'Queue',
+                        'Questions',
+                        'Feedback',
+                    ]}
+                />
             )}
             {!data.isViewerModerator && (
-                <Tabs value={tabIndex} onChange={handleTabChange}>
-                    <Tab label='Question List' />
-                    <Tab label='Live Feedback' />
-                </Tabs>
+                <Tabs 
+                    value={tabIndex}
+                    onChange={handleTabChange}
+                    tabs={[
+                        'Questions',
+                        'Feedback',
+                    ]}
+                />
```

Comment:
This could be condensed.
```jsx
const userTabs = [...];
const moderatorTabs = [...];
// ...........................
<Tabs value={...} onChange={...} tabs={isViewerModerator ? moderatorTabs : userTabs} />
```